### PR TITLE
Feature/shared template

### PR DESCRIPTION
**Pull request by hthompson-a10**
## Description
If Feature Addition:
- Required: Related user story

If Bug Fix:
- Required: Severity Level (Low, High, Critical)
- Required: Issue Description

## Jira Ticket
**This is optional if you are an external contributor**
- Required: Add links to tickets closed here

## Technical Approach
- Required: Overview of approach taken to solve the problem
- Optional: List of tasks completed w/ links 

## Config Changes
**This is only required if the config has been updated in this PR**
- Required: Provide example config snippet within pre block

<pre>
<b>Add changes in bold</b>
</pre>

(Example Snippet - Remove This Before Submission)

<pre>
[rack_vthunder]

devices=[
 {
 "project_id": "fake_uuid",
 "username" : "username",
 "password" : "password",
 "ip_address" : "10.43.12.137",
 "device_name" : "rack_1"
 <b>"example": "change",</b>
 },
 }]
</pre>

## Test Cases
- Required: List edge cases tested against in unit tests 

## Manual Testing
- Required: List of steps to manually test feature or fix

### CODE REVIEW

File: a10_octavia/controller/worker/tasks/utils.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
+import copy
```
Comment (afrin-chakure-a10) [inline a10_octavia/controller/worker/tasks/utils.py:15]: This seems to be unnecessary import

**Review by afrin-chakure-a10 — CHANGES_REQUESTED** [review]
Minor change. Otherwise all looks good.
